force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -55,10 +55,14 @@ public with sharing class FlowControlLoopsCollections {
      * @return The maximum of the two numbers, or null if either number is null.
      */
     public static Integer findMax(Integer num1, Integer num2) {
+        if (num1 == null || num2 == null) return null;
```

Comment:
This is an opinion, but it is one with good reasoning: don't make if statements without braces.

Reply:
Makes scoping unclear, and can lead to difficult to modify code.